src/coreclr/src/nativeaot/System.Private.CoreLib/src/System/Environment.CoreRT.Win32.cs

```diff
@@ -1,12 +1,16 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Diagnostics.CodeAnalysis;
 using System.Runtime.InteropServices;
 
 namespace System
 {
     public static partial class Environment
     {
+#pragma warning disable CS8763 // A method marked [DoesNotReturn] should not return.
```

Comment:
Is this needed when ExitProcess is marked DoesNotReturn? Sounds like a bug.

Reply:
Ok, it is weird that it works everywhere else. Wasn't it just an oversight?

Reply:
I'll try to submit it without this suppression, but I'm pretty sure I was getting build failure without this suppression and the DllImport is annotated.